`strip: 'bin/gcc/libgpac.so': No such file` with `--static-build`

With the following build configuration:

```
$ ./configure --prefix=$PWD/install-dir --static-build

** System Configuration
Install prefix: /home/pszemus/projects/gpac/install-dir
Source path: /home/pszemus/projects/gpac
C   compiler: gcc
C++ compiler: g++
make: make
CPU: x86_64
Big Endian: no

** GPAC 2.5-DEV rev1801-g91083fcfc-master Core Configuration **
Unit Tests: no ('make unit_tests')
Static build enabled
max stack size: 32768
debug version: no
GProf enabled: no
Memory tracking enabled: no
Sanitizer enabled: no
Fixed-Point Version: no
IPV6 Support: yes
QuickJS Support: yes (with qjs-libc)

** Detected libraries **
zlib: system (pkgconfig)
OpenGL support: no
OpenSSL support: system (pkgconfig)
nghttp2: no
ngtcp2: no
nghttp3: no
OSS Audio: system
ALSA Audio: no
Jack Audio: no
Pulse Audio: no
DirectFB: no
X11: yes (path: /usr/X11R6)
X11 Shared Memory support: no
X11 XVideo support: no
SDL: no
DVB for Linux: system
FreeType: no
JPEG: system (pkgconfig)
OpenJPEG: no
PNG: system (pkgconfig)
MAD: no
FAAD: no
XVID: no
libcaption: no
MpeghDecoder: no
libcaca: no
libcurl: no
FFmpeg: no
LZMA: yes
Xiph Vorbis: system (pkgconfig)
Xiph Theora: no
A52 (AC3): no
OpenSVCDecoder: no
OpenHEVCDecoder: no
Platinum UPnP: no
Freenect: no

Creating config.mak
Check config.log for detection failures
Done - type 'make help' for make info, 'make' to build
```

`make && make install-lib` fails with missing dynamic library:

```
$ make install-lib
[...]
make installdylib
make[1]: Wejście do katalogu '/home/pszemus/projects/gpac'
install -d "/home/pszemus/projects/gpac/install-dir/lib"
strip: 'bin/gcc/libgpac.so': No such file
make[1]: *** [Makefile:285: installdylib] Błąd 1
make[1]: Opuszczenie katalogu '/home/pszemus/projects/gpac'
make: *** [Makefile:333: install-lib] Błąd 2
```

Why `installdylib` target gets executed when `--static-build` was used?

--- a/Makefile
+++ b/Makefile
@@ -279,7 +279,7 @@ uninstall:
 
 
 installdylib:
-ifneq ($(STATIC_BINARY),yes)
+ifneq ($(STATIC_BUILD),yes)
 
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/$(lib_dir)"
 

